src/Pimple/PimpleContainerAdapter.php

```diff
@@ -48,6 +48,8 @@ public function addServiceConfig(ServiceConfig $config)
     }
 
     /**
+     * @param InflectorConfig $config
```

Comment:
> If PhpStorm doesn't know the type, add just enough documentation to inform it.

That's basically it, in a shorter form, hehe! 👍 

> The only thing I don't like about documenting types which have a typehint, is the possibility for it to be wrong. e.g.

>```php
>/**
> * @param $p Foo
> */
>function run(Bar $p) { /* ... */ }
>```

Absolutely, that's why I never add docblocks unless needed, because they tend to become outdated, if not careful!

Thank you, Tom!